root/static/scripts/edit/components/DateRangeFieldset.js

```diff
@@ -162,10 +162,12 @@ export function reducer(
 }
 
 component _DateRangeFieldset(
+  children?: React.Node,
   disabled: boolean = false,
   dispatch: (ActionT) => void,
-  endedLabel: string,
+  endedLabel?: string,
   field: DatePeriodFieldT,
+  hideEnded?: boolean = false,
```

Comment:
What is the reason for having two properties here?
Can’t `endedLabel` be optional and `hideEnded` not added?

Reply:
Just hide the whole ended section unless you pass a label? I guess that could work.